app/views/tutorials/index_teacher.html.erb

```diff
@@ -0,0 +1,109 @@
+<% if @assignments.present? %>
```

Comment:
`tutorials/index_teacher.html.erb` and `tutorials/index.html.erb` share a lot of code. I think it is advisable to refactor, e.g. by putting stuff that they both use into partials.